source/agora/network/Manager.d

```diff
@@ -1035,6 +1035,7 @@ public class NetworkManager
                             {
                                 try
                                 {
+                                    log.trace("getMissingBlockSigs: updating header signature: {} validators: {}", header.signature, header.validators);
```

Comment:
Won't the nominator log this ? In any case, unrelated to the commit.

Reply:
This is logged during catch up and not during nomination. I'll split it into a different commit.